examples/demo-vendoring/atmos.yaml

```
@@ -19,3 +19,12 @@
 logs:
   file: "/dev/stderr"
   level: Info
+
+# Custom CLI commands
+
+# No arguments or flags are required
+commands:
+- name: "test"
```

yamllint

[error] wrong indentation: expected 2 but found 0

(indentation)